amundsen_application/static/js/index.tsx

```diff
@@ -43,6 +44,7 @@ ReactDOM.render(
             <Route path="/browse" component={BrowsePage} />
             <Route path="/search" component={SearchPage} />
             <Route path="/user/:userId" component={ProfilePage} />
+            <Route path="/preferences" component={PreferencesPage} />
```

Comment:
At the moment since we might not actually get this working end-to-end this week, can you add a `TODO: Remove PreferencesPage before merging to master` in this file so that we don't forget? The reason is that we wouldn't want to expose a page that won't do anything yet.